gallery/src/styles/NxList/NxListMultiLineExample.html

```diff
@@ -6,27 +6,35 @@
     distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
 
 -->
-<div class="nx-list">
-  <h4 class="nx-list__title">
-    List title
-  </h4>
-  <ul class="nx-list">
-    <li class="nx-list__item">
-      Item 1
-      <p class="nx-list__subtext">This is a second line.</p>
-    </li>
-    <li class="nx-list__item">
-      Item 2
-      <p class="nx-list__subtext">
-        This is a second line. It includes text that might relate to the top line or might not.
-      </p>
-    </li>
-    <li class="nx-list__item">
-      Item 3
-      <p class="nx-list__subtext">
-        A long line of subtext that wraps. Add <code class="nx-code">.nx-truncate-ellipsis</code>
-        to the <code class="nx-code">&lt;p&gt;</code> tag if you don't want the text to wrap.
-      </p>
-    </li>
-  </ul>
-</div>
+<h3 class="nx-h3">
+  Examples of wrapping and truncation in lists
+</h3>
+<ul class="nx-list">
+  <li class="nx-list__item">
+    <span class="nx-list__text">
+      Text wrapping - text that is very long and will wrap onto the next line, standard list item text that is very
+      long and will wrap onto the next line
+    </span>
+  </li>
+  <li class="nx-list__item">
+    <span class="nx-list__text nx-truncate-ellipsis">
+      Text truncation - Add <code class="nx-code">.nx-truncate-ellipsis</code> to
+      <code class="nx-code">.nx-list__item</code> to cause the text to truncate at the right edge of the list item.
+    </span>
+  </li>
+  <li class="nx-list__item">
+    <span class="nx-list__text">Subtext wrapping</span>
+    <p class="nx-list__subtext">
+      A long line of subtext that wraps. Add <code class="nx-code">.nx-truncate-ellipsis</code>
+      to the <code class="nx-code">&lt;p&gt;</code> tag if you don't want the text to wrap. More text to trigger the
+      wrapping
+    </p>
+  </li>
+  <li class="nx-list__item">
+    <span class="nx-list__text">Subtext truncation</span>
+    <p class="nx-list__subtext nx-truncate-ellipsis">
+      Add <code class="nx-code">.nx-truncate-ellipsis</code> to <code class="nx-code">.nx-list__subtext</code> to
+      cause the text to truncate at the standard maximum paragraph width.
```

Comment:
I believe we've had discussions in the past that the paragraph width should only ever be used for wrapping - truncating at an arbitrary width doesn't make sense, it should only happen when the space constraints are hit.

Reply:
> I believe we've had discussions in the past that the paragraph width should only ever be used for wrapping - truncating at an arbitrary width doesn't make sense, it should only happen when the space constraints are hit.

In IQ paragraph width is used for truncation on sub-titles, that's why I did it here, but I have no issue with changing it.

Reply:
> > I believe we've had discussions in the past that the paragraph width should only ever be used for wrapping - truncating at an arbitrary width doesn't make sense, it should only happen when the space constraints are hit.
> 
> In IQ paragraph width is used for truncation on sub-titles, that's why I did it here, but I have no issue with changing it.

https://github.com/sonatype/sonatype-react-shared-components/pull/141/commits/68446a6e5fd7846bda4b38d58f16434b4d1b6ac8